sharktank/sharktank/models/clip/clip.py

```diff
@@ -487,6 +487,19 @@ def get_input_embeddings(self) -> nn.Module:
     def set_input_embeddings(self, value):
         self.text_model.embeddings.token_embedding = value
 
+    def sample_inputs(self, batch_size: int) -> OrderedDict[str, AnyTensor]:
+        return OrderedDict(
+            [
+                (
+                    "input_ids",
```

Comment:
Is empty better than some random values here? Probably doesn't matter either way, but all zeros causes bigger differences on some models iirc.